src/cascadia/TerminalControl/TermControl.h

```diff
@@ -31,8 +31,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         static Control::TermControl NewControlByAttachingContent(Control::ControlInteractivity content, const Microsoft::Terminal::Control::IKeyBindings& keyBindings);
 
-        winrt::fire_and_forget UpdateControlSettings(Control::IControlSettings settings);

```

Comment:
nit: we may want to switch to `const&` for parameters when we go from coroutine to flat method. We had to take parameters by copy due to coroutine lifetime guarantees. If we no longer need those guarantees, we could save one AddRef/Release call per call site.